examples/car_control/run.py

```diff
@@ -0,0 +1,136 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+import argparse
+import numpy as np
+import os
+import spear
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    args = parser.parse_args()
+
+    np.set_printoptions(linewidth=200)
+
+    # load config
+    config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
+    spear.configure_system(config)
+    instance = spear.Instance(config)
+
+    with instance.begin_frame():
+        gameplay_statics_uclass = instance.unreal_service.get_static_class(class_name="UGameplayStatics")
+        controller_uclass = instance.unreal_service.load_class(class_name="UObject", outer=0, name="/Script/Engine.Controller")
+        bp_car_uclass = instance.unreal_service.load_class(class_name="UObject", outer=0, name="/Game/VehicleTemplate/Blueprints/OffroadCar/OffroadCar_Pawn.OffroadCar_Pawn_C")
+        vehicle_movement_component_class = instance.unreal_service.load_class(class_name="UObject", outer=0, name="/Script/ChaosVehicles.ChaosVehicleMovementComponent")
```

Comment:
Don't need `vehicle_movement_component_class`. Either get the movement component by type (and register the type ahead of time), or better yet, get it by name.

Reply:
fixed